packages/client/src/views/UpcomingClosingDates.vue

```diff
@@ -137,8 +140,8 @@ export default {
       await this.setup();
     },
     upcomingItems() {
-      // https://lukashermann.dev/writing/how-to-use-async-await-with-vuejs-components/
       this.formatUpcoming();
+      this.formatDate();
```

Comment:
Is this needed? `formatDate` expects a parameter and appears intended to be called from BootstrapVue's Table component as a cell formatter.